templates/HelpFiles/InstructorSendMail.html.ep

```diff
@@ -17,19 +17,43 @@
 % title maketext('Email Help');
 %
 <p>
-	<%= maketext('Emails can be sent from this page to the class. Be sure to change all blanks which read "fixme". The '
-		. 'email can be sent to all students, or some subset. To change the way the students are ordered, after '
-		. 'selecting the criteria, push the "Change Display Settings" button.') =%>
+	<%= maketext('Use this page to send emails to active (enrolled or auditing) students.  Emails can be sent to '
+		. 'all active students or selected students.  Use the "Students" form to sort, filter, and/or format how '
```

Comment:
Please change "and/or" to just "or".  Instructors of courses that use WeBWorK (mostly mathemeticians) know that "or" can be inclusive.

Reply:
Done. Thanks.